Sources/SwiftProtobufTestHelpers/Descriptor+TestHelpers.swift

```diff
@@ -9,9 +9,8 @@
 // -----------------------------------------------------------------------------
 
 import SwiftProtobuf
-import SwiftProtobufPluginLibrary
 
-extension Google_Protobuf_FileDescriptorProto {
+public extension Google_Protobuf_FileDescriptorProto {
```

Comment:
The other option was to just copy this file into both Tests/* directories, but using a shared target seems slightly better.